database/factories/MemberFactory.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use Faker\Generator as Faker;
+
+/** @var $factory Closure */
+$factory->define(Proto\Models\Member::class,
+    function (Faker $faker) {
+        $picktime = $faker->dateTimeInInterval('April 20, 2011', 'now');
+        return [
+            'proto_username' => strtolower(str_random(16)),
+            'created_at' => $faker->dateTime($picktime)->format('Y-m-d H:i:s'),
+            'deleted_at' => (mt_rand(0, 1) === 1 ? null : $faker->dateTimeBetween($picktime, '+1 year')->format('Y-m-d H:i:s')),
+            'is_lifelong' => mt_rand(0, 20) > 15 ? 1 : 0,
```

Comment:
Maybe you could use these ratios here: https://www.proto.utwente.nl/queries/membership_totals

Reply:
I made some approximations based on these ratios.